### PR TITLE
:sparkles: Adding a Swap Page

### PR DESCRIPTION
**Pull request by KipandaJr**

### CODE REVIEW

File: views/swap/manage-slippage/manage-slippage-form/index.tsx [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+            <Typography variant="label" size="large" width="100%">
+              Normal
+            </Typography>
+          </Button>
+          <Button
+            variant="outline"
+            textAlign="center"
+            onClick={() => setSpeed('fast')}
+          >
+            <Typography variant="label" size="large" width="100%">
```
Comment (MarioBatalha) [inline views/swap/manage-slippage/manage-slippage-form/index.tsx:111]: Why are u putting typography inside of button?

File: views/swap/input/index.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+const Input: FC<InputProps> = ({ label, formSwap }) => {
+  const balance = useWatch({
+    control: formSwap.control,
+    name: `${label}.balance`,
+  });
+
+  return (
+    <Box
+      border="1px solid"
+      borderColor="outlineVariant"
```
Comment (MarioBatalha) [inline views/swap/input/index.tsx:21]: In layout on Figma there are no borders in card balance.

File: views/swap/input/header-info.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import { HeaderInfoProps } from './input.types';
+
+const HeaderInfo: FC<HeaderInfoProps> = ({ label, balance }) => (
+  <Box px="l" display="flex" justifyContent="space-between">
+    <Typography variant="label" size="small">
+      {label}
+    </Typography>
+    <Typography variant="label" size="small">
+      Balance:
+      <Typography variant="label" size="small" color="primary" as="span">
```
Comment (MarioBatalha) [inline views/swap/input/header-info.tsx:13]: Is missing right border in the balance value

File: views/swap/input/index.tsx (hunk 2 above)
Comment (MarioBatalha) [reply views/swap/input/index.tsx:21]: Change border size and border color in exchange box .

File: views/swap/input/dropdown-token/item.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+};
+
+const DropdownTokenItem: FC = () => {
+  return (
+    <Motion
+      alignItems="center"
+      variants={itemVariants}
+      initial={itemVariants.closed}
+      justifyContent="space-between"
+      cursor={'pointer'}
```
Comment (MarioBatalha) [inline views/swap/input/dropdown-token/item.tsx:22]: You can put as simple string cursor="pointer"

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

File: views/swap/manage-slippage/manage-slippage-form/index.tsx (hunk 1 above)
Comment (KipandaJr) [reply views/swap/manage-slippage/manage-slippage-form/index.tsx]: to use the variant about Typography

**Review by okusindja — COMMENTED** [review]
Reviewed. Need only to update the Collapsible container and dropdown with the updated version of UI-kit

**Pull request merged**